### ISSUE TITLE
Loss of ** function calling in 0.7.7

### ISSUE THREAD

**Issue by jdavisp3** [gruns/orderedmultidict#11]
The use of orderedmultidicts in `**` kwargs function invocations seems to have stopped working with `0.7.7`. This code now prints out an empty dict:

```
def print_kwargs(**kwargs): print kwargs

import orderedmultidict

d = orderedmultidict.omdict({'a': 1, 'b': 2})

print_kwargs(**d)
```

With `0.7.6` it prints `{'a': 1, 'b': 2}`.

**Comment by gruns** [comment]
Fix now live in orderedmultidict v0.7.8.


```python
>>> import orderedmultidict
>>> orderedmultidict.__version__
0.7.8
>>> o = orderedmultidict.omdict([('a', 'a'), ('b', 'b')])
>>> splat = lambda *args, **kwargs: (args, kwargs)
>>> splat(*o, **o)
(('a', 'b'), {'a': 'a', 'b': 'b'})
```

If you use orderedmultidict via furl, the latest furl v0.5.6 depends on the
newly minted orderedmultidict v0.7.8. Upgrade with
`pip install orderedmultidict -upgrade` or `pip install furl --upgrade`.

Thank you for bringing this issue to my attention Dave. Don't hesitate to let me
know if there's anything else I can do for you.

**Comment by jdavisp3** [comment]
Thanks so much @gruns !
